readme.md

```diff
@@ -191,24 +194,88 @@ Print is the only language defined i/o function. It takes in a single variable a
 The following are valid print statements:
 ```
 print ( "hello" );
-print ( a );
+print ( 42 );
+print ( a );  # Only valid if the variable "a" is defined.
 ```
 
 ### Loops
 A loop will itterate between a range of numbers.
+
 Example loops:
 ```
-loop ( a in range, ( 1, 100 ) ) {
+loop ( a in range( 1, 100 ) ) {
 
 	print ( a );
 
 }
 ```
+### Type casting
+Most data types can be converted into any other data type.
+```
+# Convert to int.
+# Can be done with all data types.
+int(x)
+
+# Convert to float.
+# Can be done with all data types.
+float(x)
+
+# Convert to string.
+# Can be done with all data types.
+string(x)
+
+# Convert to bool.
+# Can be done with all data types.
+bool()
+
+# Convert to none.
+# Can be done with all data types.
+none(x)
```

Comment:
Why is none needed? Isn't it just setting it equal to none?

Reply:
It is useless, but now you can convert any data type to any other data type.

Reply:
Then none wouldn't be a great name for it, since none implies no data, also can't you use just use the other cast functions?

Reply:
None is a weird type since it signifies the lack of a type rather than a specific set of data. This is likely valid but unnecessary since none is only useful internally to the interpreter